### PR TITLE
docs: wrap up link fixes [APE-1482]

### PR DESCRIPTION
**Pull request by antazoey**
### What I did

The real issue was fixed by https://github.com/ApeWorX/ape/pull/1321
but it was never closed
This completes it by auditing and "fixing" the rest: fixes: #1263 

I could find no more bad links after this

### Checklist

- [ ] All changes are completed
- [ ] New test cases have been added
- [ ] Documentation has been updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ The CLI tool contains all the Ape commands and the Python SDK contains the class
 
 ## **Ape Modular Plugin System:**
 
-Our [list of plugins](https://www.apeworx.io/#plugins) is the best way to have the most interoperable experience with Web3.
```
Comment (antazoey) [inline README.md:83]: this is no longer a real link it seems

File: docs/userguides/clis.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ This guide is for showcasing utilities that ship with Ape to assist in your CLI
 
 ## Ape Context Decorator
 
-The `@ape_cli_context` gives you access to all the root Ape objects (`accounts`, `networks` etc.), the ape logger, and an `abort` method for stopping execution of your CLI gracefully.
+The [@ape_cli_context](../methoddocs/cli.html#ape.cli.options.ape_cli_context) gives you access to all the root Ape objects (`accounts`, `networks` etc.), the ape logger, and an [abort](../methoddocs/cli.html#ape.cli.options.ApeCliContextObject.abort) method for stopping execution of your CLI gracefully.
```
Comment (antazoey) [inline docs/userguides/clis.md:21]: these were just missed opportunities for links

File: docs/userguides/testing.md [hunk 3]
```diff
@@ -47,17 +47,12 @@ To disable isolation add the `--disable-isolation` flag when running `ape test`
 
 ## Fixtures
 
-Fixtures are any type of reusable instances of something with configurable scopes. `pytest` handles passing fixtures
-into each test method as test-time. To learn more about [fixtures](https://docs.pytest.org/en/7.1.x/explanation/fixtures.html)
+You can define and use `pytest` fixtures in your Ape tests.
+Learn more about fixtures from [this guide](https://docs.pytest.org/en/7.1.x/explanation/fixtures.html).
+The syntax and functionalities of fixtures work exactly the same in Ape as it does with `pytest`.
 
-Define fixtures for static data used by tests. This data can be accessed by all tests in the suite unless specified otherwise. This could be data as well as helpers of modules which will be passed to all tests.
```
Comment (antazoey) [inline docs/userguides/testing.md:53]: this part was just wordy and distracting, so i tidied it up

**Review by fubuloubu — APPROVED** [review]

File: src/ape/types/signatures.py [hunk 4]
```diff
@@ -7,6 +7,11 @@
 
 from ape.types import AddressType
 
+# Fix 404 in doc link.
```
Comment (antazoey) [inline src/ape/types/signatures.py:10]: This is maybe the 404 the original ticket was talking about?
Before, we just got the doc from the package it is from (in the EF), but its links didnt work when in ape's docs

This fixes it!

File: src/ape/api/accounts.py [hunk 5]
```diff
@@ -59,7 +59,7 @@ def sign_message(self, msg: SignableMessage) -> Optional[MessageSignature]:
         Sign a message.
 
         Args:
-          msg (:class:`~ape.types.signatures.SignableMessage`): The message to sign.
+          msg (:class:`~ape.types.signatures.   SignableMessage`): The message to sign.
```
Comment (fubuloubu) [inline src/ape/api/accounts.py:62]: Why this have a space?
```suggestion
          msg (:class:`~ape.types.signatures.SignableMessage`): The message to sign.
```

**Review by fubuloubu — APPROVED** [review]

File: src/ape/api/accounts.py (hunk 5 above)
Comment (antazoey) [reply src/ape/api/accounts.py]: i wish there was a fat-finger emoji

**Pull request merged**